util/util_test.go

```diff
@@ -22,7 +21,7 @@ func TestPrintableUTF8OrEmpty(t *testing.T) {
 		},
 		{
 			"Asset 510285544",
-			"8J+qmSBNb25leQ==",
```

Comment:
Why did this need to change?

Reply:
Not sure of the exact root cause, but from what I can tell, that string is `🪙 Money, want` which returns a printable string (failing the test which expects it not to be printable) while the new string `🪙 Money, want\n` includes newline and is not printable (as anticipated) on the newline character.

It looks like the most recent update to the unicode bits in go is https://github.com/golang/go/commit/8ec5a052ec8d4e628353fb5099b29341fae9b3a4 which started in 1.16. That could be the root cause if we were expecting the moon symbol to fail since we're on 1.13. But I haven't dug that deep on it.

Reply:
This also reminded me that I forgot to update golang in this PR. Updating with that.

Reply:
https://github.com/algorand/indexer/pull/943 also includes changes to this test so it must be failing on golang1.16 as well.